database/migrations/2022_02_08_143200_books.php

```diff
@@ -13,12 +13,12 @@ class Book extends Migration
      */
     public function up()
     {
-        Schema::create('book', function (Blueprint $table) {
+        Schema::create('books', function (Blueprint $table) {
             $table->id();
-            $table->string('book_name');
-            $table->integer('book_ibsn');
+            $table->string('name');
+            $table->bigInteger('isbn');
```

Comment:
bu kolonun integer olması çok da doğru değil aslında, neden? Çünkü isbn numarası aslında bir sayı niteliğinde değildir. Aynı şekilde veritabanında bu konu telefon numaraları için de geçerlidir. 

Eğer sıfır ile başlayabilecek değerler söz konusu ise, tutulacak değer gerçekten bir sayıdan ibaret değilse, string tutulmalıdır.

Çünkü sen değer olarak 0345242238 girip kaydetmeye çalışırsan veritabanında kaydedilecek değerin farklı olacağını görebilirsin.

aynı şekilde telefon numarası gibi 05354305675 girersen veritabanında kaydedilmiş halinin, senin girdiğin değer olmadığını görebilirsin.

isbn numarası iki formata sahiptir. 13 haneli ve 10 haneli olabiliyor. 10 haneliler 0 ile başlayabiliyor mesela https://en.wikipedia.org/wiki/International_Standard_Book_Number